src/main/java/com/group12/degreeaudit/Student.java

```diff
@@ -3,20 +3,56 @@
 import java.util.List;
 
 public class Student {
-    public String name;
-    public String ID;
-    public String program;
-    public String semesterAdmitted;
-    public List<Course> courses = new ArrayList<Course>();
+    private String name;
+    private String ID;
+    private String program;
+    private String semesterAdmitted;
+    private List<Course> coursesTaken = new ArrayList<Course>();
 
-    public Student() {
+    public Student() {}
+    public Student(String name, String ID, String program, String semesterAdmitted, List<Course> coursesTaken) {
+        this.name = name;
+        this.ID = ID;
```

Comment:
All caps variables might not be the best naming convention. ID is also registering weird